Unable to successfully complete OMR

**Describe the bug**
OMR fails during dewarp

**Input Image**
The original image that triggers this bug
<img src="https://github.com/BreezeWhite/oemer/assets/68708352/4fe9de86-1492-4b6d-93b1-a5af3c1e9d86" width="200px" />
<img src="https://github.com/BreezeWhite/oemer/assets/68708352/2a19ea75-efad-4823-a669-311ecc32a38b" width="200px" />

**Full Traceback**
```
Traceback (most recent call last):
  File "/opt/homebrew/bin/oemer", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/oemer/ete.py", line 276, in main
    mxl_path = extract(args)
               ^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/oemer/ete.py", line 146, in extract
    coords_x, coords_y = estimate_coords(staff)
                         ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/oemer/dewarp.py", line 271, in estimate_coords
    new_gg_map = connect_nearby_grid_group(gg_map, grid_groups, grid_map, grids)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/oemer/dewarp.py", line 186, in connect_nearby_grid_group
    assert grid_id in grid_groups[label].gids, f"{grid_id}, {label}"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: -1, -1
```

Adding `--without-deskew` I get the following Traceback
```
Traceback (most recent call last):
  File "/opt/homebrew/bin/oemer", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/oemer/ete.py", line 276, in main
    mxl_path = extract(args)
               ^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/oemer/ete.py", line 167, in extract
    staffs, zones = staff_extract()
                    ^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/oemer/staffline_extraction.py", line 327, in extract
    rr = np.array(rr, dtype=np.int)
                            ^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/numpy/__init__.py", line 305, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations. Did you mean: 'inf'?
``` 

Running with tensorflow `oemer music.png --use-tf -d` I get the following Traceback
```
Traceback (most recent call last):
  File "/opt/homebrew/bin/oemer", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/oemer/ete.py", line 276, in main
    mxl_path = extract(args)
               ^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/oemer/ete.py", line 129, in extract
    os.environ["INFERENCE_WITH_TF"] = ori_inf_type
    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "<frozen os>", line 684, in __setitem__
  File "<frozen os>", line 758, in encode
TypeError: str expected, not NoneType
```

**Command You Execute**
`oemer music.png` or `oemer music.jpg` have the same trace and error

--- a/oemer/staffline_extraction.py
+++ b/oemer/staffline_extraction.py
@@ -324,7 +324,7 @@ def extract(splits=8, line_threshold=0.8, horizontal_diff_th=0.1, unit_size_diff
     all_staffs = []
     for rr in zones:
         print(rr[0], rr[-1], end=' ')
-        rr = np.array(rr, dtype=np.int)
+        rr = np.array(rr, dtype=int)
         staffs = extract_part(staff_pred[:, rr], x_offset=rr[0], line_threshold=line_threshold)
         if staffs is not None:
             all_staffs.append(staffs)

